app/controllers/github_webhooks_controller.rb

```diff
@@ -0,0 +1,36 @@
+class GithubWebhooksController < ApplicationController
+  
+  # POST /author_webhook
+  def author_webhook
+    action = params["github_webhook"]["action"]
```

Comment:
We prefer to use symbol keys for hashes instead of string keys. You could also use `dig` here to make this a little simpler:

```suggestion
    action = params.dig(:github_webhook, :action)
```

Reply:
Thanks @undergroundwebdesigns 
This is really a good practice, i use in this way from now